Resources/Prototypes/_White/tags.yml

```
@@ -1,2 +1,5 @@
 - type: Tag
-  id: EnergySword
\ No newline at end of file
+  id: EnergySword
+
+- type: Tag
+  id: NeuroStabilization
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)